ResearchKit.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+  s.name         = 'ResearchKit'
+  s.version      = '1.0.0'
+  s.summary      = 'ResearchKit is an open source software framework that makes it easy to create apps for medical research or for other research projects.'
+  s.homepage     = 'https://www.github.com/ResearchKit/ResearchKit'
+  s.documentation_url = 'http://researchkit.github.io/docs/'
+  s.license      = { :type => 'BSD', :file => 'LICENSE' }
+  s.author       = { "Apple Inc." => "http://apple.com" }
```

Comment:
Let's change author to researchkit.org, and the URL to http://researchkit.org .
Although Apple was the initial owner, and we continue to be involved, this is now going to be a community project.